docs-src/references/strongly-typed-errors/cause-unhandled-update.md

```diff
@@ -0,0 +1,17 @@
+---
+id: cause-unhandled-update
+title: Cause Unhandled Update
+description: Explanation for Cause Unhandled Update error message, and how to fix it.
+sidebar_label: Cause Unhandled Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+`UnhandledUpdate` occurs when the [Workflow Execution](/concepts/what-is-a-workflow-execution) attempts to close itself before handling pending updates.
```

Comment:
Need clarification what a "pending" update means here.
I'll come up with something more accurate here.

Reply:
@bergundy any additional thoughts here? would like to close this it.